x-pack/plugins/data_visualizer/public/application/index_data_visualizer/utils/get_supported_aggs.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { DataViewField } from '@kbn/data-views-plugin/common';
+
+export const isCounterTimeSeriesMetricField = (field: DataViewField) =>
+  field.timeSeriesMetric === 'counter';
+
+const SUPPORTED_AGGS = {
+  COUNTER: new Set(['count', 'min', 'max']),
+  AGGREGATABLE: new Set(['count', 'cardinality', 'percentiles', 'stats', 'terms']),
+  DEFAULT: new Set<string>(),
+};
+
+/**
+ * Temporarily add list of supported ES aggs until the PR below is merged
+ * https://github.com/elastic/elasticsearch/pull/93884
+ */
+export const getSupportedAggs = (field: DataViewField) => {
+  if (isCounterTimeSeriesMetricField(field)) {
+    return SUPPORTED_AGGS.COUNTER;
```

Comment:
Discover are using different icons for `counter` and `gauge` time series metrics. Is that something we can do in the data visualizer? One for a follow-up perhaps?

<img width="409" alt="image" src="https://user-images.githubusercontent.com/7405507/228551091-964e6754-8c37-4d50-b240-2a671749a220.png">

Reply:
This is now synced up with Discover/Unified field list here [`8b504d5` (#153893)](https://github.com/elastic/kibana/pull/153893/commits/8b504d5d7ba98dd828a545627a1f95c3d5786e74)